espnet/asr/pytorch_backend/asr.py

```diff
@@ -196,7 +196,7 @@ def update_core(self):
         if self.grad_noise:
             from espnet.asr.asr_utils import add_gradient_noise
             add_gradient_noise(self.model, self.iteration, duration=100, eta=1.0, scale_factor=0.55)
-        loss.detach()  # Truncate the graph
+        loss.detach_()  # Truncate the graph
```

Comment:
I do not think this line is needed because `loss` is not referenced below.

Reply:
Yes, you are right.